rust/datafusion/src/execution/physical_plan/udf.rs

```diff
@@ -84,6 +86,19 @@ impl ScalarFunctionExpr {
     }
 }
 
+impl fmt::Display for ScalarFunctionExpr {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}(", self.name)?;
+        for i in 0..self.args.len() {
```

Comment:
`self.args.join(", ")`?

Reply:
Or, the complete statement, `write!(f, "{}({})", self.name, self.args.join(", "))?;`

Reply:
That wouldn't work but I pushed this change instead:

```rust
        write!(
            f,
            "{}({})",
            self.name,
            self.args
                .iter()
                .map(|e| format!("{}", e))
                .collect::<Vec<String>>()
                .join(", ")
        )
```

Reply:
yeap, forgot that .join required strings. Still better IMO, as it is more expressive.